nightfall-optimist/src/routes/proposer.mjs

```diff
@@ -98,12 +98,14 @@ router.post('/update', async (req, res, next) => {
   logger.debug({ msg: '/update endpoint', payload: JSON.stringify(req.body, null, 2) });
 
   try {
-    const { address, url = '' } = req.body;
+    const { address, url = '', fee = 0 } = req.body;
     if (url === '') {
```

Comment:
I am wondering why do we do this check `if (url === '') { … }` here but we don't do it in the block above?

Reply:
Yes, I agree with you. If the `url` should be provided always and different from '' it should be provided also and checked the same in `register` and `update` proposer.